docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,144 @@
+
+## Generate an Octopus Tentacle PFX File {#ClusteringTentacles-Newpfx}
+
+Open `cmd` as Administrator again and run these commands to generate a new Private Key from an Octopus Deploy Tentacle (replacing relevant values as appropriate).
+```batchfile
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe new-certificate --export-pfx="<sharedStorageDriveLetter>:\TentacleClusterPrivateKey.pfx" --pfx-password="Yourpfxpassword"
+```
+
+## Import the new Octopus Tentacle PFX File {#ClusteringTentacles-Importpfx}
+
+Now import the new pfx file into the server from which it was just generated.
+```batchfile
+Tentacle.exe import-certificate --instance="Tentacle" --from-file="<sharedStorageDriveLetter>:\TentacleClusterPrivateKey.pfx" --pfx-password="Yourpfxpassword"
+Tentacle.exe service --instance="Tentacle" --stop --start
+```
+
+Now on the second node, stop the Tentacle service and bring the shared storage offline the same way you have done before. This time go back to the first node in the cluster, bringing the Shared Storage drive back online and start the tentacle service. Then, import the pfx file into the first node to ensure both nodes in the cluster hold the same private key. 
+
+Once both Tentacles are installed and configured ensure that neither node has the Octopus Tentalce started and that the shared storage is brought offline.
+
+## Configure A new Clustered Service {#ClusteringTentacles-NewCluster}
+
+Ensure each node that will be participating in the Tentacle Cluster is joined to the Active Directory Domain and has the **Failover Clustering** feature installed in Windows. For more information on installing the Failover Clustering feature in windows follow [this link](https://blogs.msdn.microsoft.com/clustering/2012/04/06/installing-the-failover-cluster-feature-and-tools-in-windows-server-2012/ "installing the failover cluster service feature and toold in windows server 2012")
+
+Open the **Failover Cluster Manager** console on one of the nodes. If there is no cluster configured yet, you can right click **Failover Cluster Manager** amd select **New Cluster**
+
+On the **Select Servers** page, enter the Fully Qualified Domain Name of each node that will be in this cluster.
+
+On the **Validation Warning** page, select `Yes, When I click Next, run configuration validation tests, and then return to the process of creating the cluster`
+
+The **Validate a Configuration Wizard** appears, select `Run all Tests` and select **Next**
+
+When all validation processes successfully, you will be returned to the **Create Cluster Wizard** where the **Access Point for Administering the Cluster** page appears. At this point, choose an IP Address that is on the same Network as both Tentacles and a hostname that is 15 characters or less. 
```

Comment:
If you change the previous para to start with `When`, then this one will need to change to something else as well. Maybe `After all validation processes complete successfully`?